src/rpc/blockchain.cpp

```diff
@@ -1328,7 +1329,7 @@ static RPCHelpMan verifychain()
                 "\nVerifies blockchain database.\n",
                 {
                     {"checklevel", RPCArg::Type::NUM, RPCArg::DefaultHint{strprintf("%d, range=0-4", DEFAULT_CHECKLEVEL)},
-                        strprintf("How thorough the block verification is:\n - %s", Join(CHECKLEVEL_DOC, "\n- "))},
```

Comment:
In commit "refactor: Reuse MakeUnorderedList where possible" (bb56486a170aacb355f4a973f0cd40ab3918a0cd)

This looks good but just want to note small change in behavior here: space is removed before the first "-"

Reply:
Yes. While working on this PR I was wondering what is the purpose of a space before the first "-" :)

It seems ignored:
```
$ src/bitcoin-cli --version
Bitcoin Core RPC client version v22.99.0-bb56486a170a
$ src/bitcoin-cli help verifychain > ~/pr
$ src/bitcoin-cli --version
Bitcoin Core RPC client version v22.99.0-03826aecc56c
$ src/bitcoin-cli help verifychain > ~/master
$ diff ~/master ~/pr  # empty
```